compat/mingw.c

```diff
@@ -3296,8 +3587,32 @@ static void setup_windows_environment(void)
 			struct strbuf buf = STRBUF_INIT;
 			strbuf_addstr(&buf, tmp);
 			if ((tmp = getenv("HOMEPATH"))) {
```

Comment:
Everything before this line is entirely unrelated to the commit message

Reply:
> what's the purpose of an automatic code formatter in the codebase if it's not going to be used?

Not Git for Windows' choice. It's Git's choice. Complain to them.